config/constants.ts

```diff
@@ -63,8 +77,8 @@ export const eventBusName = 'OrcaBusMain';
 export const eventSourceQueueName = 'orcabus-event-source-queue';
 
 // Event Archiver constants for EventBus Contruct in SharedStack
-export const archiveBucketName = 'event-archive-bucket';
-export const archiveSecurityGroupName = 'OrcaBusEventArchiveSecurityGroup';
+export const archiveBucketNamePrefix = 'orcabus-event-archive-';
```

Comment:
Hm, a quite generic name for a specific purpose bucket. There may be other archive buckets besides the event archive... 

Also, I am not sure this bucket is meant to be for anything else than internal use. Nobody else needs to know that name and it does not have to be configurable. As such we could move its definition into the stack (or even construct) level?

Reply:
I move the Event Archiver definition from into the EventBus Stack level config, as event bus users may need config archiver